camel/typing.py

```diff
@@ -24,10 +24,13 @@ class RoleType(Enum):
 
 
 class ModelType(Enum):
-    GPT_3_5_TURBO = "gpt-3.5-turbo"
-    GPT_3_5_TURBO_16K = "gpt-3.5-turbo-16k"
+    GPT_3_5_TURBO = "gpt-3.5-turbo-1106"
+    GPT_3_5_TURBO_16k = "gpt-3.5-turbo-1106"
```

Comment:
Sorry that there was a typo. I think, Let's capitalize `k` to `K` for `GPT_3_5_TURBO_16K` and `GPT_4_32K`.